sdk/core/azure-core/src/main/java/com/azure/core/util/CoreConstants.java

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util;
+
+/**
+ * Class to hold commonly used constants.
+ */
+public final class CoreConstants {
```

Comment:
I feel a little uneasy with making this public API right now. I would rather it be impl to start with?